src/OpenTK/Platform/Linux/LinuxJoystick.cs

```diff
@@ -71,29 +74,144 @@ internal sealed class LinuxJoystick : IJoystickDriver2
 
         private readonly object sync = new object();
 
-        private readonly FileSystemWatcher watcher = new FileSystemWatcher();
-
         private readonly DeviceCollection<LinuxJoystickDetails> Sticks =
             new DeviceCollection<LinuxJoystickDetails>();
 
         private bool disposed;
 
+        private const string InputDevicePath = "/proc/bus/input/devices";
```

Comment:
Does this work on BSD? Hipster distros? Is this a *nix standard? Is there is a syscall we can make to query this path?

Reply:
It works on BSD if the Linux procfs package is installed.
Otherwise, it should just fall back to loading once from evdev at startup (and no reconnection as currently)

Reply:
That's good enough for me!